networking/kvm.go

```diff
@@ -420,6 +424,7 @@ func kvmTransformFlannelNetwork(net *activeNet) error {
 	net.conf.Type = n.Delegate["type"].(string)
 	net.conf.IPMasq = n.Delegate["ipMasq"].(bool)
 	net.conf.MTU = n.Delegate["mtu"].(int)
+	net.conf.IsDefaultGateway = n.Delegate["isDefaultGateway"].(bool)
```

Comment:
This should be consistent with https://github.com/containernetworking/cni/blob/master/plugins/main/bridge/bridge.go#L190